subprojects/docs/src/docs/userguide/declaringDependencies.adoc

```diff
@@ -18,42 +18,53 @@
 Gradle builds can declare dependencies on external binaries, raw files and other Gradle projects. You can find examples for common scenarios in this section. For more information, see the <<dependency_types,full reference on all types of dependencies>>.
 
 [[sec:declaring_binary_dependency]]
-=== Declaring a binary dependency
+=== Declaring a dependency to a module
 
-Modern software projects rarely build code in isolation. Projects reference external libraries for the purpose of reusing existing and proven functionality, so-called _binary dependencies_. Upon resolution binary dependencies are downloaded from dedicated repositories and stored in a cache to avoid unnecessary network traffic.
+Modern software projects rarely build code in isolation. Projects reference external modules for the purpose of reusing existing and proven functionality. Upon resolution, selected versions of external modules are downloaded from dedicated repositories and stored in a cache to avoid unnecessary network traffic.
 
 +++++
 <figure xmlns:xi="http://www.w3.org/2001/XInclude">
-    <title>Resolving binary dependencies from remote repositories</title>
+    <title>Resolving dependencies from remote repositories</title>
     <imageobject>
         <imagedata fileref="img/dependency-management-binary-dependencies.png" width="150mm" />
```

Comment:
:x: We should rename the image file as well here.

Reply:
done